src/Security/Security/Models/SqlVulnerabilityAssessment/PSSqlVulnerabilityAssessmentRule.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.SecurityCenter.Models.SqlVulnerabilityAssessment
+{
+    public class PSSqlVulnerabilityAssessmentRule
+    {
+        public string RuleId { get; set; }
+        public string Severity { get; set; }
+        public string Category { get; set; }
+        public string RuleType { get; set; }
+        public string Title { get; set; }
+        public string Description { get; set; }
+        public string Rationale { get; set; }
+        public PSSqlVulnerabilityAssessmentQueryCheck QueryCheck { get; set; }
+        public IList<PSSqlVulnerabilityBenchmarkReference> BenchmarkReferences { get; set; }
+
+        public override string ToString()
+        {
+            return $"{{\r\n" +
+                $"Rule id:      {RuleId}\r\n" +
+                $"Severity:     {Severity}\r\n" +
```

Comment:
Please replace \r\n with Environment.NewLine to be compatible in different platforms.

Reply:
pending